src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,24 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../MovieType';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(({
+      title, description, imgUrl, imdbUrl, imdbId,
+    }) => (
+      <MovieCard
+        key={imdbId}
+        title={title}
+        description={description}
+        imgUrl={imgUrl}
+        imdbUrl={imdbUrl}
+      />
+    ))}
```

Comment:
it wouldhave the same effect 

```suggestion
    {movies.map((movie) => (
      <MovieCard key={imdbId} {..movie} />
    ))}
```